iree/compiler/Conversion/HLOToLinalg/BUILD

```diff
@@ -51,35 +51,21 @@ cc_library(
 cc_library(
     name = "HLOToLinalg",
     srcs = [
-        "HLOToLinalgOnBuffers.cpp",
-        "Passes.cpp",
         "ResolveShapeOps.cpp",
```

Comment:
Interesting! I think `ResolveShapeOps.cpp` can go too, but it is used for now, but probably not needed. Not needed to be resolved here, just a note for (my) future reference.

Reply:
I tried to remove it, but it's used for old lit tests. I plan to delete it later.
And it also broke regressiondynamic_add.mlir, but I did not dig into it.